docker-compose.yml

```diff
@@ -18,13 +18,14 @@ services:
     depends_on:
       - db
   app: &app
-    image: ubcctlt/ipeer-app
+    ##image: ubcctlt/ipeer-app
```

Comment:
could you revert this change back? If you need to use your local container, you can just add `--build` in the docker compose command.